QualityControl/docs/Testing.md

```diff
@@ -0,0 +1,31 @@
+# Project Testing
+
+## 1. Install dependencies
+Before running the tests, ensure that all project dependencies are installed.
+Run the following command in the project root directory:
+```bash
+npm install
```

Comment:
In the installation step, you recommend using `npm ci`. Is there a reason, here you recommend `npm install` instead?